src/mono/browser/build/BrowserWasmApp.targets

```diff
@@ -260,6 +260,8 @@
       <EmccCompileOptimizationFlag Condition="'$(EmccCompileOptimizationFlag)' == ''">$(_EmccOptimizationFlagDefault)</EmccCompileOptimizationFlag>
       <!-- FIXME: which one should take precedence? -->
       <WasmCompileOptimizationFlag>$(EmccCompileOptimizationFlag)</WasmCompileOptimizationFlag>
+      <WasmBitcodeCompileOptimizationFlag Condition="'$(EmccLinkOptimizationFlag)' == '' and '$(Configuration)' == 'Release'">-O2</WasmBitcodeCompileOptimizationFlag>
+      <WasmBitcodeCompileOptimizationFlag Condition="'$(EmccLinkOptimizationFlag)' == '' ">WasmCompileOptimizationFlag</WasmBitcodeCompileOptimizationFlag>
```

Comment:
```suggestion
      <WasmBitcodeCompileOptimizationFlag Condition="'$(EmccLinkOptimizationFlag)' == '' ">$(WasmCompileOptimizationFlag)</WasmBitcodeCompileOptimizationFlag>
```

Reply:
This logic looks quite complicated and I don't love that we appear to be hardcoding -02 everywhere for release instead of changing the default for release and still allowing overrides

Reply:
I agree. Note that we have different flags for different parts, we also have `EmccCompileOptimizationFlag` ending with `-Oz` for Release.

This fix aims to quickly fix the regression.